doc/sketch.md

```diff
@@ -0,0 +1,83 @@
+This documents intends to give an overview of the time complexity of the different graph transformations implemented by the various data types existent in Alga.
+
+When specifying the time and memory complexity of graph algorithms, _n_ will denote the number of vertices in the graph, _m_ will denote the number of edges in the graph, and _s_ will denote the _size_ of the corresponding `Graph` expression.
+
+**`removeVertex`**: Removes a vertex from a given graph. In `NonEmpty` graphs, returns `Nothing` if the resulting graph is empty.
+
+| removeVertex       | Time          | Memory |
```

Comment:
The first header here should probably be `Data type` instead of `removeVertex` (same for other tables).